stable/nginx-ingress/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -13,7 +13,10 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 spec:
+  {{- if .Capabilities.APIVersions.Has "batch/v1alpha1" -}}
```

Comment:
This shouldn't have the extra `-` at the end.
This should be:
```
{{- if .Capabilities.APIVersions.Has "batch/v1alpha1" }}
```

Reply:
Thanks!